docs/configuration/config-file-reference.md

```diff
@@ -1031,6 +1031,12 @@ The `query_range_config` configures the query splitting and caching in the Corte
 # CLI flag: -querier.split-queries-by-interval
 [split_queries_by_interval: <duration> | default = 0s]
 
+# Split metadata APIs by an interval and execute in parallel, 0 disables it. You
+# should use an a multiple of 24 hours (same as the storage bucketing scheme),
+# to avoid queriers downloading and processing the same series.
+# CLI flag: -querier.split-metadata-by-interval
+[split_metadata_by_interval: <duration> | default = 0s]
```

Comment:
The name metadata usually includes `series` and other 2 labels API. If we only support series first then let's mention it here? Other labels API are not supported now.

Reply:
The pan is to immediately implement this for the label names and label values APIs as well. I didn't want to do it in a single PR to keep the size in check.

I can add this to the doc.